alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -248,22 +250,29 @@ class MonitorRunner(
                     if (it.value.throttled) 1 else 0) })
         }
 
+        if (alertSchemaVersion == null) {
+            alertSchemaVersion = SchemaVersionUtils.getSchemaVersion(AlertIndices.alertMapping())
+        }
+
         // Merge the alert's error message to the current alert's history
         val updatedHistory = currentAlert?.errorHistory.update(alertError)
         return if (alertError == null && !result.triggered) {
             currentAlert?.copy(state = COMPLETED, endTime = currentTime, errorMessage = null,
-                    errorHistory = updatedHistory, actionExecutionResults = updatedActionExecutionResults)
+                    errorHistory = updatedHistory, actionExecutionResults = updatedActionExecutionResults,
+                    schemaVersion = alertSchemaVersion!!)
```

Comment:
Can we avoid using `!!` here? and on line 269/275?

Reply:
Will change it